pages/Profile/index.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import Layout from "../../components/Layout/Layout";
+import styles from "./Profile.module.css";
+
+const ProfileSettingsPage: React.FunctionComponent = () => {
+    
+    return (
+        <Layout title = 'Profile'>
+            <main className = {styles.main}>
+                <h1 className = {styles.title}>
```

Comment:
`h1` (header tags) are usually for just displaying text and they actually can't contain children/nested elements within them. We use flex containers/flexbox for elements that can contain other children/nested elements so we can apply the layout/spacing we want. So in this case, you shouldn't apply the style for a flexbox directly on the header, but rather on a div or other element that contains the header.